packages/public/@babylonjs/viewer-alpha/package.json

```diff
@@ -36,7 +36,8 @@
         "@rollup/plugin-terser": "^0.4.4",
         "@rollup/plugin-typescript": "^11.1.6",
         "rollup": "^4.18.0",
-        "rollup-plugin-dts": "^6.1.1"
+        "rollup-plugin-dts": "^6.1.1",

```

Comment:
if lit is not needed as a dev dependencies here, I assume it is because it is all built from the other package and in this case why do we need the rest of the dev tools ?

Reply:
Oh, actually this is a miss. Lit needs to be a regular dependency or it will be missing when the consumer installs this package. Fixed and tested locally.

The public package does actually rebuild and bundle the code (differently for the package vs. the test tools like size analysis), which is why it also needs the same dev dependencies like rollup plugins.